pkg/apis/servicecatalog/types.go

```diff
@@ -226,8 +226,41 @@ type ServiceClass struct {
 	AlphaRequires []string
 }
 
+type Plan struct {
+	// PlanRef is a reference to the full plan details.
+	PlanRef v1.LocalObjectReference `json:"planRef"`
```

Comment:
I think this might be related something I don't think we resolved... when I GET a serviceclass how much of the Plan is pulled back with it? If there are 300 plans do we want to force the UI/client to do 300 GETs too?  See https://github.com/kubernetes-incubator/service-catalog/issues/454#issuecomment-301197304